sgid_client/decrypt_data.py

```diff
@@ -0,0 +1,32 @@
+from jwcrypto import jwk, jwe
+from .error import Errors
+
+
+def decrypt_data(encrypted_key: str, encrypted_data: dict, private_key: str):
+    try:
```

Comment:
does it matter if we're not exporting this function?

Reply:
ah sorry, misunderstood what you meant, will fix this in a bit

Reply:
fixed this in [8aa2c5a](https://github.com/opengovsg/sgid-client-python/pull/2/commits/8aa2c5a8de9d38bd364ff88da01a9d58b41bb814) and [b9c76a9](https://github.com/opengovsg/sgid-client-python/pull/2/commits/b9c76a9cc441f932535a9b62d152584c156e1781).

I realised that in the NodeJS SDK, while we do a try-catch when importing the private key, an error can also be thrown earlier in `convertPkcs1ToPkcs8` (and this error is not caught) if the private key starts with the PKCS1 header but is invalid. in Python, rather than having the same error potentially thrown in two different places, I've placed the try-catch in the `convert_pkcs1_to_pkcs8` function which is always called on the private key, so any potential errors are caught there. also added a corresponding test for this in [b3f9d98](https://github.com/opengovsg/sgid-client-python/pull/3/commits/b3f9d98844d95d5f9993309ace375b9a59980c95).